index.js

```diff
@@ -0,0 +1,40 @@
+// tooling
+import postcss from 'postcss';
+
+// plugin
+export default postcss.plugin('postcss-system-ui-font', opts => {
+	const familyOpt = Object(opts).family;
+
+	const systemUiFamily = typeof familyOpt === 'string'
+		? familyOpt.trim().split(/\s*,\s*/)
+	: familyOpt || [
+		'system-ui',
+		'-apple-system',
+		'BlinkMacSystemFont',
+		'Segoe UI',
+		'Roboto',
+		'Oxygen',
+		'Ubuntu',
+		'Cantarell',
+		'Fira Sans',
+		'Droid Sans',
+		'Helvetica Neue'
+	];
+
+	// system-ui and fallbacks match
+	const whitespace = '[\\n\\r\\x09\\x20]';
+	const systemUiMatch = new RegExp(`(^|,|${whitespace})(?:${whitespace}*system-ui${whitespace}*)(?:,${whitespace}*(?:${systemUiFamily.join('|')})${whitespace}*)?(,|$)`, 'i');
```

Comment:
When `system-ui` is used as a fallback generic font, the replace phrase will remove the extra comma space.
Expected:
```
"Droid Sans", system-ui, foo
```
Actual
```
"Droid Sans",system-ui, foo
```

Reply:
I will add a test for this and fix it. Thanks!